credentials/oauth/oauth.go

```diff
@@ -66,27 +66,28 @@ func (ts TokenSource) RequireTransportSecurity() bool {
 }
 
 type jwtAccess struct {
-	ts oauth2.TokenSource
+	jsonKey []byte
+	//ts oauth2.TokenSource
 }
 
-func NewJWTAccessFromFile(keyFile string, audience string) (credentials.Credentials, error) {
+func NewJWTAccessFromFile(keyFile string) (credentials.Credentials, error) {
 	jsonKey, err := ioutil.ReadFile(keyFile)
 	if err != nil {
 		return nil, fmt.Errorf("credentials: failed to read the service account key file: %v", err)
 	}
-	return NewJWTAccessFromKey(jsonKey, audience)
+	return NewJWTAccessFromKey(jsonKey)
 }
 
-func NewJWTAccessFromKey(jsonKey []byte, audience string) (credentials.Credentials, error) {
-	ts, err := google.JWTAccessTokenSourceFromJSON(jsonKey, audience)
+func NewJWTAccessFromKey(jsonKey []byte) (credentials.Credentials, error) {
+	return jwtAccess{ jsonKey }, nil
```

Comment:
yeah, I should. I ran gofmt -w **/*.go but did not notice this file is 2-level deeper and did not get touched.